### PR TITLE
Merge Jetpack Compose 1.6.0

### PR DESCRIPTION
**Pull request by igordmn**

### CODE REVIEW

File: libraryversions.toml [hunk 1]
```diff
@@ -19,8 +19,8 @@ CAMERA_PIPE = "1.0.0-alpha01"
 CAMERA_TESTING = "1.0.0-alpha01"
 CARDVIEW = "1.1.0-alpha01"
 CAR_APP = "1.7.0-alpha01"
-COLLECTION = "1.4.0-rc01"
-COMPOSE = "1.6.0-rc01"
+COLLECTION = "1.4.0"
+COMPOSE = "1.6.0"
 COMPOSE_COMPILER = "1.5.4"
```
Comment (MatkovIvan) [inline libraryversions.toml:24]: Should it be 1.5.8?

File: libraryversions.toml [hunk 2]
```diff
@@ -19,8 +19,8 @@ CAMERA_PIPE = "1.0.0-alpha01"
...
 COMPOSE_MATERIAL3 = "1.2.0-alpha11"
```
Comment (MatkovIvan) [inline libraryversions.toml:25]: Do we already have 1.2.0 alpha in jb-main?

**Review by MatkovIvan — APPROVED** [review]

File: libraryversions.toml (hunk 1 above)
Comment (igordmn) [reply libraryversions.toml:24]: It is 1.5.4 in the 1.6.0 commit.

But we don't use the compiler from this commit, we use it from maven (and publish separately from another branch)

File: libraryversions.toml (hunk 2 above)
Comment (igordmn) [reply libraryversions.toml:25]: No, it is still 1.1.2. This version isn't used. And we don't keep it in sync with the compose/material3 folder

**Pull request merged**